docs/site/Discovering-models.md

```diff
@@ -35,7 +35,7 @@ Models can be discovered from a supported datasource by running the
 
 ### Options
 
-`--dataSource`: Put a valid datasource name here to skip the datasource prompt
+`--datasource`: Put a valid datasource name here to skip the datasource prompt
```

Comment:
Rather than replacing the old prompt, can we not support both old and new. This will ensure backward compatibility. Otherwise it will break existing cli command.